scenes/game/game_wrapper.gd

```diff
@@ -37,24 +37,35 @@ func initialize_local_game(player_deck, opponent_deck, randomize_first_player):
 	var first_player = Enums.PlayerId.PlayerId_Player
 	if randomize_first_player and randi() % 2 == 0:
 		first_player = Enums.PlayerId.PlayerId_Opponent
-	current_game.initialize_game(player_deck, opponent_deck, "Player", "CPU", 
-		first_player, seed_value)
+	current_game.initialize_game(player_deck, 
+		opponent_deck, 
+		"Player", 
+		"CPU", 
+		first_player, 
+		seed_value)
 	current_game.draw_starting_hands_and_begin()
 
-func initialize_remote_game(player_info, opponent_info, starting_player : Enums.PlayerId, 
-	seed_value : int, observer_mode : bool, starting_message_queue : Array):
+func initialize_remote_game(player_info, 
+		opponent_info, 
+		starting_player : Enums.PlayerId, 
+		seed_value : int, 
+		observer_mode : bool, 
+		starting_message_queue : Array):
 	current_game = RemoteGame.new()
-	current_game.initialize_game(player_info, opponent_info, starting_player, 
-		seed_value, observer_mode, starting_message_queue)
+	current_game.initialize_game(player_info, 
+		opponent_info, 
+		starting_player, 
+		seed_value, 
+		observer_mode, 
+		starting_message_queue)
 
 # Deletes the current game
 func end_game():
 	current_game.free()
 	current_game = null
 
-# Triggers a timeout
-func trigger_timeout(losing_player : Enums.PlayerId, reason : Enums.GameOverReason):
-	current_game.trigger_timeout(losing_player, reason)
+func submit_clock_ran_out():
+	current_game.submit_clock_ran_out()
```

Comment:
this should really be current_game.do_clock_ran_out() to mirror every other function